src/db/spanner/models.rs

```diff
@@ -1247,13 +1247,19 @@ impl SpannerDb {
         if let Some(timestamp) = offset.clone().unwrap_or_default().timestamp {
             query = match sort {
                 Sorting::Newest => {
-                    sqlparams.insert("older_eq".to_string(), as_value(timestamp.as_rfc3339()?));
-                    sqltypes.insert("older_eq".to_string(), as_type(TypeCode::TIMESTAMP));
+                    sqlparams.insert(
```

Comment:
I figured it made sense to leave these changes. The commented-out code was still using the now non-existent `as_value` function